src/App.js

```diff
@@ -1,16 +1,33 @@
-import React, { useState } from "react"
-import "./App.css"
-import LightSwitch from "./components/LightSwitch"
+import React, { useState } from "react";
+import "./App.css";
+import LightSwitch from "./components/LightSwitch";
+
 
 const App = () => {
-  const [lightSwitchStatus, setLightSwitchStatus] = useState("off")
 
   return (
-    <>
-      <h1>Lightbulb Challenge</h1>
-      <LightSwitch />
```

Comment:
You can use the old line 11 \<LightSwitch /> over and over again to keep creating new boxes. This method uses a lot less code and is more dynamic. I tried it with your existing code and it works. The only difference is the direction the boxes are rendered on screen.